### DISCUSSION TITLE
Scrolling is jumpy (not smooth)

### DISCUSSION THREAD

**Discussion by raster** [pixelfed/pixelfed-rn#33] [Bug Reports]
When scrolling on an iPhone SE (2nd Gen) it is not smooth and jumps around. (I tried to upload a video showing this but got an error. Let me know if you want to see it.)

**Comment by ghost** [comment]
Got the same issue on iPhone 15+
Here is a screen capture during slow scrolling:
https://share.icloud.com/photos/02dT7GloogCZi3ex10WtfOf8g

The faster I scroll the bigger the jumps seem.
